app/controllers/concerns/announce_actions/announce_filter.rb

```diff
@@ -65,6 +65,7 @@ def decorator_for_model(model)
 
     def decorator_class(name)
       "Api::Msg::#{name}Representer".safe_constantize ||
+        "Api::Auth::#{name}Representer".safe_constantize ||
```

Comment:
Just to sanity check - seem to be no implications for existing announcements here.  Looks like `Auth::Series` and `Auth::Story` just rewrite some links, which won't impact anything.